internal/goreleaser/goreleaser.go

```diff
@@ -4,10 +4,10 @@
 package goreleaser
 
 import (
-	"fmt"
+	"bytes"
+	"html/template"
```

Comment:
Do we want `text/template` here? `html/template` does some escaping, which might not be desired in the YAML output.

Reply:
Yeah, we don't need escaping